backend/src/server.cpp

```diff
@@ -31,7 +31,14 @@ namespace server
         std::function <std::string(sqlite3_stmt *)> callback = [](sqlite3_stmt *stmt) {
             return std::string(reinterpret_cast<const char *>(sqlite3_column_text(stmt, 0)));
         };
-        std::vector<std::string> breached_passwords = db.execute("SELECT * FROM passwords;", callback);
+
+        // get the table num corresponding to the user password leaked byte
+        std::string decoded_password = crow::utility::base64decode(user_password, user_password.size());
```

Comment:
**nitpick**: could we use `decoded_password` in the `encryptPassword` function below?